src/SDK/Logs/LoggerProviderInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Logs;
+
+use OpenTelemetry\API\Logs as API;
+
+interface LoggerProviderInterface extends API\LoggerProviderInterface
+{
```

Comment:
**question:** Are there other functions you'd like to add besides `shutdown()` and `forceFlush()` here?

Reply:
no, I think those are the only interface methods that the SDK adds... note that this interface extends the API interface, which is also how we do it for tracing.